contrib/completion/bash/docker

```diff
@@ -5572,11 +5581,8 @@ _docker() {
 		wait
 	)
 
-	local known_plugin_commands=()
-
-	if [ -f "$COMPOSE_PLUGIN_PATH" ] ; then
-		known_plugin_commands+=("compose")
-	fi
+	PLUGINS_INSTALLED=$(docker info --format '{{range .ClientInfo.Plugins}}{{ .Name }} {{end}}')
```

Comment:
question: shouldn't we have a curated list here instead?

Reply:
I don't think so. Every plugin should have the posibility to hook into completion.

Reply:
Perhaps; otoh, if the plugin is there, adding completion for it doesn't really harm I guess.
We should consider having a `v0.0.2` version of the metadata that indicates that the plugin supports the `__complete` subcommands (although current code _should_ handle it gracefully).

Reply:
Actually; currently that won't work, because we have the dedicated `_docker_<plugin name>` functions 🤦